common/src/warp_runner/ui_adapter/message_event.rs

```diff
@@ -147,6 +149,18 @@ pub async fn convert_message_event(
         } => MessageEvent::ConversationNameUpdated {
             conversation: messaging.get_conversation(conversation_id).await?,
         },
+        MessageEventKind::MessagePinned {
```

Comment:
I'd prefer the new enumerations be `MessageEventKind::Pinned` and `MessageEventKind::Unpinned` or  `MessageEventKind::MessagePinned` and `MessageEventKind::MessageUnpinned`. The latter is redundant but the `MessageReactionRemoved/Added` events have that format too. (maybe they should be changed too)

Reply:
idk about that. since every other enum value follows the same principle. or do you mean `MessageEvent` instead of `MessageEventKind`?

Reply:
I stand by what I wrote. Including the part about the other enums following the same naming pattern.

Reply:
the thing is though for `MessageEventKind` all commands that deal with message have `Message` prefixed.
`MessageSent, MessageReceived ... etc`. so should they all be changed?

should i then update the other enums also? or should that be in a separate pr?

Reply:
I think it would be better to rename them to remove the `Message` prefix.  Adding that there was my mistake. Probably should be done in a separate PR just to make this PR easier to read.

Reply:
It can be done in a separate PR for now if it havent been done in this PR. I remember the reasoning for me having the `Message` prefix in the enum was due to some conflicts earlier on when streams were introduced, which looking over the code isnt applicable now so it may change in the future on warp end but not exactly priority.

Reply:
i will then for now adjust the enum related to this pr. and the other enums can be changed in another pr